app/assets/stylesheets/main.scss

```diff
@@ -34,6 +34,11 @@ $path: '/static/images/';
 // Dependencies from GOV.UK Frontend, packaged to be specific to this application
 @import './govuk-frontend/all';
 
+// Custom overrides
+.govuk-link {
```

Comment:
Where is this being used?

Reply:
The sms-message-wrapper??   Here is the error.  You'll see ultimately the 'target' is govuk-link, but it doesn't point to a page or anything:


Accessibility violation: {'id': 'color-contrast', 'impact': 'serious', 'tags': ['cat.color', 'wcag2aa', 'wcag143'], 'description': 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds', 'help': 'Elements must have sufficient color contrast', 'helpUrl': 'https://dequeuniversity.com/rules/axe/4.4/color-contrast?application=axeAPI', 'nodes': [{'any': [{'id': 'color-contrast', 'data': {'fgColor': '#0066CC', 'bgColor': '#DFE1E2', 'contrastRatio': 4.24, 'fontSize': '12.0pt (16px)', 'fontWeight': 'normal', 'messageKey': None, 'expectedContrastRatio': '4.5:1', 'shadowColor': None}, 'relatedNodes': [{'html': '

', 'target': ['.sms-message-wrapper']}], 'impact': 'serious', 'message': 'Element has insufficient color contrast of 4.24 (foreground color: #0066CC, background color: #DFE1E2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1'}], 'all': [], 'none': [], 'impact': 'serious', 'html': '[Notify.gov](http://notify.gov/)', 'target': ['.govuk-link'], 'failureSummary': 'Fix any of the following:\n Element has insufficient color contrast of 4.24 (foreground color: #0066CC, background color: #DFE1E2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1'}]}

Reply:
Ok this looks like it's just in one spot. Approved.